yt/utilities/on_demand_imports.py

```diff
@@ -204,6 +204,36 @@ def create(self):
 
 _pooch = pooch_imports()
 
+_pooch = pooch_imports()
```

Comment:
oops, looks like a merge or copy/paste mistake here. will let tests finish before fixing this...  
```suggestion
_pooch = pooch_imports()
```